django/boss/settings/production.py

```diff
@@ -57,25 +58,42 @@
     SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
     SESSION_CACHE_ALIAS = 'default'
 
+# Set this here so it's not overriden by any other settings files.
+LOGIN_URL = BOSSOIDC_LOGIN_URL
+LOGOUT_URL = BOSSOIDC_LOGOUT_URL
+
 INSTALLED_APPS.append("bossoidc")
-INSTALLED_APPS.append("djangooidc")
+INSTALLED_APPS.append("mozilla_django_oidc")
 INSTALLED_APPS.append("rest_framework.authtoken")
-AUTHENTICATION_BACKENDS.insert(1, 'bossoidc.backend.OpenIdConnectBackend')
 
 REST_FRAMEWORK['DEFAULT_AUTHENTICATION_CLASSES'] = (
+    'mozilla_django_oidc.contrib.drf.OIDCAuthentication',
     'rest_framework.authentication.SessionAuthentication',
     'boss.authentication.TokenAuthentication',
     'oidc_auth.authentication.BearerTokenAuthentication',
 )
 
+AUTHENTICATION_BACKENDS.insert(1, 'bossoidc.backend.OpenIdConnectBackend') 
+
 auth_uri = vault.read('secret/endpoint/auth', 'url')
 client_id = vault.read('secret/endpoint/auth', 'client_id')
 public_uri = vault.read('secret/endpoint/auth', 'public_uri')
 
+OIDC_OP_AUTHORIZATION_ENDPOINT = auth_uri + '/protocol/openid-connect/auth'
+OIDC_OP_TOKEN_ENDPOINT = auth_uri + '/protocol/openid-connect/token'
+OIDC_OP_USER_ENDPOINT = auth_uri + '/protocol/openid-connect/userinfo'
+LOGIN_REDIRECT_URL = public_uri + 'v1/mgmt'
+LOGOUT_REDIRECT_URL = auth_uri + '/protocol/openid-connect/logout?redirect_uri=' + public_uri
+OIDC_RP_CLIENT_ID = client_id
+OIDC_RP_CLIENT_SECRET = ''
+OIDC_RP_SCOPES = 'sub preferred_username'
+OIDC_RP_SIGN_ALGO = 'RS256'
+OIDC_OP_JWKS_ENDPOINT = auth_uri + '/protocol/openid-connect/certs'
```

Comment:
So how do you figure out you needed to add all these OIDC values?  Are they from mozilla_django_oidc?

Reply:
The docs and experimentation. I had to run everything locally to figure this out.